landa/organization_management/doctype/landa_member/landa_member.json

```diff
@@ -293,9 +254,14 @@
   {
    "link_doctype": "Yearly Fishing Permit",
    "link_fieldname": "member"
+  },
+  {
+   "custom": 1,
```

Comment:
```suggestion
```

Not sure what this does, but it leads to a double entry in connections for me. 
![image](https://user-images.githubusercontent.com/79863208/204783142-d952df02-b3b3-443e-b8ed-65687ac36fae.png)
Removing this line reduces it to one link in the connections that works just fine.